pkg/baseline/baseline.props

```diff
@@ -5,5 +5,13 @@
     <!-- Set PackageIndexFile to point to our local repo's index.  This tells the packaging targets
          which index should be used for updates -->
     <PackageIndexFile>$(MSBuildThisFileDirectory)..\Microsoft.Private.PackageBaseline\packageIndex.json</PackageIndexFile>
+    
+    <!-- set the project.json file for pkgproj's to point to the project.json in this directory
+         to ensure that PKGProjs will restore harvested packages. -->
+    <ProjectJson>$(MSBuildThisFileDirectory)\project.json</ProjectJson>
+    <ProjectLockJson>$(MSBuildThisFileDirectory)\project.lock.json</ProjectLockJson>
   </PropertyGroup>
+
+  <!-- make RestorePackages run before HarvestStablePackage to ensure we'll restore the baseline before harvesting  -->
+  <Target Name="EnsureRestore" BeforeTargets="HarvestStablePackage" DependsOnTargets="RestorePackages" />
```

Comment:
I'm a little worried about the performance of adding this extra restore to every pkgproj but if you do the other item and make RestorePackages incremental then it won't be a big issue.